pages/common/zpool.md

```diff
@@ -4,30 +4,28 @@
 
 - Show the configuration and status of all ZFS zpools:
 
-`zpool status [{{poolname}}]`
+`zpool status`
 
 - Check a ZFS pool for errors (verifies the checksum of EVERY block). Very CPU and disk intensive:
 
-`zpool scrub {{poolname}}`
+`zpool scrub {{pool_name}}`
 
 - List zpools available for import:
 
 `zpool import`
 
-- Import a zpool, optionally specifying a new name:
+- Import a zpool (optionally renaming it):
 
-`zpool import {{poolname}}`
-`zpool import {{poolname}} {{newpoolname}}`
+`zpool import {{pool_name}} {{[new_pool_name]}}`
```

Comment:
Yeah, it's tough, because you can be much more expressive without requiring adding a whole extra example.  It also breaks copy paste (both cause you have to remove brackets and decide whether you want that option).

If we decide to allow this I'd imagine a lot of examples would have a `[-v]` which wouldn't be terrible (again a compact way to express things) but would certainly require it's on style guide mention: "if possible put all optional flags at the end so as to simplify copy/paste for the common use"